src/aks-preview/azext_aks_preview/managed_cluster_decorator.py

```diff
@@ -3151,6 +3158,11 @@ def update_network_plugin_settings(self, mc: ManagedCluster) -> ManagedCluster:
         pod_cidr = self.context.get_pod_cidr()
         if pod_cidr:
             mc.network_profile.pod_cidr = pod_cidr
+
+        network_policy = self.context.get_network_policy()
+        if network_policy:
+            mc.network_profile.network_policy = network_policy
```

Comment:
in addition to the live test you added, should we have a unit test in test_managed_cluster_decorator.py? Maybe add something to `test_update_network_plugin_settings`?

Reply:
Added a test case to update network-policy from empty string to "azure"

Reply:
Python implicitly converts `""` to `False`, if user specifies `--network-policy ""`, the code will not update the property value to `""`, is this expected?